ci/code_checks.sh

```diff
@@ -56,7 +56,7 @@ if [[ -z "$CHECK" || "$CHECK" == "lint" ]]; then
     black --version
 
     MSG='Checking black formatting' ; echo $MSG
-	black . --check --exclude '(asv_bench/env|\.egg|\.git|\.hg|\.mypy_cache|\.nox|\.tox|\.venv|_build|buck-out|build|dist|setup.py)'
+	black . --check
```

Comment:
Moved the `--exclude` option to `pyproject.toml` since I had to create an entry there for `target-version`.  Also has the advantage that users running `black` (either explicitly or through `pre-commit`) will now pick this up as well, so local behavior will match CI.